Gemfile.lock

```diff
@@ -127,6 +139,7 @@ GEM
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
+    sqlite3 (1.3.11)
```

Comment:
You have duplicates of sqlite3 and also bcrypt.  Was there a reason for it or was that a mistake?